scss/edx/overrides/_modals.scss

```diff
@@ -4,6 +4,6 @@
 }
 
 .modal-footer {
-  padding-bottom: 1.75rem;
+  padding-bottom: 0.75rem;
```

Comment:
the edX theme causes more spacing on the bottom of the modal footer than should be there (e.g., below any buttons).

Reply:
nice. this was bothering me 😄